src/openforms/forms/api/public_api/views.py

```diff
@@ -0,0 +1,29 @@
+from django.utils.translation import gettext_lazy as _
+
+from drf_spectacular.utils import extend_schema, extend_schema_view
+from rest_framework import permissions
+from rest_framework.authentication import TokenAuthentication
+from rest_framework.generics import ListAPIView
+
+from ...models import Form
+from .filters import FormCategoryNameFilter
+from .pagination import NoPagination
+from .serializers import FormSerializer
+
+
+@extend_schema_view(
+    get=extend_schema(
+        summary=_("List forms"),
+        description=_(
+            "List the available forms with minimal information. Does not include soft deleted forms."
+        ),
+    ),
+)
+class FormListView(ListAPIView):
+    authentication_classes = (TokenAuthentication,)
+    permission_classes = (permissions.IsAdminUser,)
```

Comment:
I don't think we should require a staff user - having an API client user that is able to log in to the admin is maybe a bit too much exposure.

Instead, I would:

* require the `view_form` permission (builtin django permissions)
* create a new group in the default_groups fixture, named "Public API client", and add the necessary permissions (view forms, view categories)